packages/compiler/src/i18n/extractor_merger.ts

```diff
@@ -394,10 +394,12 @@ class _Visitor implements html.Visitor {
         const nodes = this._translations.get(message);
         if (nodes) {
           if (nodes.length == 0) {
-            translatedAttributes.push(new html.Attribute(attr.name, '', attr.sourceSpan));
+            translatedAttributes.push(
+                new html.Attribute(attr.name, '', attr.sourceSpan, null, undefined, undefined));
```

Comment:
Copying comment from https://github.com/angular/angular/pull/39027#discussion_r496181365:

nit: there are few places in compiler code where we add inline comments for arguments where it's not immediately obvious what this argument is. May be we can use this approach here (and in some places below) too?
```suggestion
                new html.Attribute(attr.name, '', attr.sourceSpan, null /* keySpan */, undefined /* valueSpan */, undefined /* i18n */));
```